### ISSUE TITLE
Fasmg compile error with VERBOSE=1

### ISSUE THREAD

**Issue by bystepii** [lantonov/asmFish#97]
Compiling with following options:
```
fasmg "x86/fish.asm" "asmFishW_popcnt.exe" -i "VERSION_OS='W'" -i "VERSION_POST = 'popcnt'" -i "USE_SPAMFILTER = 1" -i "USE_WEAKNESS = 1" -i "USE_VARIETY = 1" -i  "USE_MATEFINDER = 1" -i "VERBOSE = 1" -e 1000
```
Getting this error:
```
flat assembler  version g.i17sx
x86/fish.asm [184] x86/Position.asm [540]:
        @@: xor esi, 56
macro @@ [0]
Processed: @@: xor esi, 56
Error: extra characters on line.
x86/fish.asm [184] x86/Position.asm [540]:
        @@: xor esi, 56
macro @@ [2]:
        label tail
Processed: xor esi
Error: missing argument.
x86/fish.asm [192] x86/PrintParse.asm [441]:
        @@: test rsi, rsi
macro @@ [0]
Processed: @@: test rsi, rsi
Error: extra characters on line.
x86/fish.asm [192] x86/PrintParse.asm [441]:
        @@: test rsi, rsi
macro @@ [2]:
        label tail
Processed: test rsi
Error: missing argument.
x86/fish.asm [192] x86/PrintParse.asm [444]:
        blsr rsi, rsi, rax
macro blsr [0]
Processed: blsr rsi, rsi, rax
Error: extra characters on line.
````

**Comment by MichaelB7** [comment]
Do you get errors when using make.bat?

**Comment by bystepii** [comment]
@MichaelB7 Only when I turn option VERBOSE=1
  Without it works fine

**Comment by tthsqe12** [comment]
fixed in https://github.com/tthsqe12/asm/commit/e79ce474b299efc0a5f8ed8bf12fe3cf43d1303a

**Comment by lantonov** [comment]
Also merged into e79ce47
